aten/src/ATen/native/EmbeddingBag.cpp

```diff
@@ -5,6 +5,8 @@
 
 #include <TH/THBlasUtils.h>
 
+#include <caffe2/perfkernels/embedding_lookup.h>
```

Comment:
How does the dispatch work? Keep in mind that this file isn't compile with avx.

Reply:
It's a similar mechanism to aten/DISPATCH and does the right thing with different compiler options and runtime cpuid dispatch between them: https://github.com/pytorch/pytorch/blob/master/caffe2/perfkernels/common.h

we should bring them together, but it's for another day